en/docs/apis/api-concepts.md

```diff
@@ -1,23 +1,69 @@
 # API Concepts
-Before you start managing APIs via Choreo, there are basic concepts that you need to understand. This section walks you through all the concepts you need to know. 
+Choreo's API management capabilities allow you to create APIs, manage APIs, and expose services and external APIs as managed APIs. Here are some basic concepts which will help you understand API management in Choreo as you create and manage your APIs in the [Choreo Console]({{choreo_console}}).
  
 ## API
-An API in Choreo allows you to expose business functionality as a service to customers, partners, suppliers, and other stakeholders.
+An API is an intermediate layer that acts as a communication protocol between a consumer and a service, simplifying the consumption of the service. In addition to hiding the underlying implementation details of a service, an API provides a secure, controlled, and well-documented approach to access an exposed service.
 
-## API Management Components
-Described below are the key components of Choreo API management:
+## API format
+OpenAPI Specification (OAS, a.k.a Swagger) format is the underlying representation of an API in Choreo. You can import an API using a Swagger definition in the Choreo console. Additionally, a user can view, edit, import, or download an API definition in OAS format via the Choreo console.
 
-### API Publisher 
-The API Publisher is designed for API creators to develop, document, secure, test, and version APIs with ease.
+## API resource path and HTTP methods
+An API comprises one or more resources, each of which has a unique resource path (URI). An API Resource has a set of HTTP methods that operates on it. The supported HTTP methods are `GET`, `POST`, `PUT`, `DELETE`, `PATCH`, `HEAD`, and `OPTIONS`.
```

Comment:
Since we use `URI` within brackets shouldn't we use the exact name i.e. `Unique Resource Identifire` instead of `unique resource path`

Reply:
On second thoughts, spelling out URI is not actually needed. https://developers.google.com/style/abbreviations